acinclude.m4

```diff
@@ -214,60 +214,22 @@ AC_DEFUN([MK_AM_REQUIRE_C99], [
   fi
 ])
 
-AC_DEFUN([MK_AM_REQUIRE_CXX11], [
+AC_DEFUN([MK_AM_REQUIRE_CXX14], [
```

Comment:
Switching to C++14, now supported by compilers. [Debian testing is currently using g++ 6.3(https://packages.debian.org/search?keywords=gcc&searchon=names&suite=all&section=all)], where [C++14 is the default compiler mode when compiling c++](https://gcc.gnu.org/projects/cxx-status.html#cxx14).